dev-haskell/hw-kafka-client/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>haskell@gentoo.org</email>
+		<name>Gentoo Haskell</name>
+	</maintainer>
+	<use>
+		<flag name="it">Run integration tests</flag>
```

Comment:
I'm happy to pull this in as-is, however we should either disable this flag or subsume it under USE=test (if the integration tests are appropriate to run in src_test()).

Reply:
I'd disable it, yes